SwissTransferFeatures/SettingsView/EditSettings/EditSettingsDatasource.swift

```diff
@@ -0,0 +1,189 @@
+/*
+ Infomaniak SwissTransfer - iOS App
+ Copyright (C) 2024 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import InfomaniakDI
+import STCore
+import STResources
+import SwiftUI
+import SwissTransferCore
+import SwissTransferCoreUI
+
+/// Something used to populate a `EditSettingsView`
+protocol EditCellModel {
+    var label: String { get }
+    var action: () -> Void { get }
+    var leftIconAsset: STResourcesImages? { get }
+}
+
+struct EditCellDatom: EditCellModel, Hashable, Equatable {
```

Comment:
`EditCellDatom` as a name feels weird, maybe we can find something else ?

Reply:
Yeah, figured I'd get that comment.

This name is the notion of an atom of immutable data. Only thing that came to mind not wanting to not use some UIKit ish naming scheme. A leftover from the PoC.
That's something from an [obscure](https://docs.datomic.com/datomic-overview.html) database.

I'll find something else.